packages/core-web/src/styles/scroll-top-button.css

```diff
@@ -0,0 +1,19 @@
+#scroll-top-button {
+    --scroll-top-button-opacity: 40%;
```

Comment:
would removing the variable be clearer? since its only going to be used in one element

Reply:
may also be causing this (netlify preview, chrome)

![scrolltopbutton](https://user-images.githubusercontent.com/3306138/92262501-d37cb480-ef0d-11ea-8ec5-463adca598bc.gif)

Reply:
This was resolved by changing percentage values to its represented decimal values!